test/com/google/enterprise/adaptor/filenet/FileNetAdaptorTest.java

```diff
@@ -97,15 +109,16 @@ private static DocId newDocId(String id) {
     return new DocId("guid/" + id);
   }
 
+  @Ignore // TODO(bmj): this test is broken wrt UserContext assertions
```

Comment:
`@Ignore` takes a string argument that is printed by "ant test", so you can just include the comment (TODO and all) in the annotation: `@Ignore("TODO(bmj): ...")`